pkg/ovs/ovsconfig/interfaces.go

```diff
@@ -26,14 +26,15 @@ const (
 
 	OVSDatapathSystem OVSDatapathType = "system"
 	OVSDatapathNetdev OVSDatapathType = "netdev"
+
+	OVSOtherConfigDatapathIDKey string = "datapath-id"
 )
 
 type OVSBridgeClient interface {
 	Create() Error
 	Delete() Error
 	GetExternalIDs() (map[string]string, Error)
 	SetExternalIDs(externalIDs map[string]interface{}) Error
-	GetDatapathID() (string, Error)
```

Comment:
why removing this method but keeping SetDatapathID from the interface? The struct still has both methods.